docs/user-guide/binned-data/binned-data.ipynb

```diff
@@ -231,23 +231,32 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "jupyter": {
+     "source_hidden": true
```

Comment:
Why is this needed?

Reply:
It hides the cell. As plopp is not ready to replace the old plotting I wanted to keep information about it out of the docs.

Reply:
Ah, I misread, not sure how this slipped in, will remove.